src/__mocks__/api-handlers.js

```diff
@@ -66,36 +71,36 @@ export const handlers = [
           download_url: downloadURL,
           type: 'file',
           content: 'U2hhcmUgaXMgYSB3ZWItYmFzZWQgQklNICYgQ0FEIGludGVncmF0aW9uIGVu\n' +
-              'dmlyb25tZW50IGZyb20gW2JsZHJzLmFpXShodHRwczovL2JsZHJzLmFpLyku\n' +
```

Comment:
There's still this large block of whitespace changes -- can you revert this?

Reply:
done